src/components/typography.js

```diff
@@ -1,54 +1,50 @@
 import React from 'react';
-import {
-  StyleSheet,
-  Text,
-} from 'react-native';
+import {StyleSheet, Text} from 'react-native';
 
-export const H1 = (props) => {
-    return <Text style={styles.h1}>{props.children}</Text>;
-}
+export const H1 = props => {
```

Comment:
Só uma dica, com arrow function, se for um simples retorno de uma linha, vc pode não colocar as chaves e o return

```jsx
export const H1 = props => (<Text style={styles.h1}>{props.children}</Text>);
```